pkg/etcdenvvar/etcd_env.go

```diff
@@ -334,3 +352,8 @@ func getEtcdEndpoints(configmapLister corev1listers.ConfigMapLister, skipBootstr
 
 	return strings.Join(etcdURLs, ","), nil
 }
+
+func convertDBSize(dbSize int64) string {
```

Comment:
added this to convert to gigibytes ... also added test cases

Reply:
maybe call it GibibytesToBytesString? this seems more broadly applicable than DB sizes. Did you check whether there's some utility in the stdlib that does similar conversion?

Reply:
there are third party libs, not stdlib